app/models/manageiq/providers/openstack/cloud_manager/provision/cloning.rb

```diff
@@ -4,7 +4,7 @@ def do_clone_task_check(clone_task_ref)
       instance = openstack.handled_list(:servers).detect { |s| s.id == clone_task_ref }
       status   = instance.state.downcase.to_sym
 
-      return true if status == :active
+      return true if [:active, :error].include?(status)
```

Comment:
well yeah, this true means continue in state machine, so it should theoretically contain all finite states, which in deploying should be only active or error http://docs.openstack.org/developer/nova/vmstates.html

Without this, the automate will get stuck actually, which is pretty bad, cause then you wait like hour for timeout(timeout of the whole request, other vms will get deployed) While you know almost instantly nothing will change, so no reason to wait.